extensions/wikia/PageHeader/PageHeaderController.class.php

```diff
@@ -2,9 +2,15 @@
 
 namespace Wikia\PageHeader;
 
+
 class PageHeaderController extends \WikiaController {
 
 	public function index() {
+		global $wgTitle;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)